app/views/contexts/questionnaire_summary_context.py

```diff
@@ -34,5 +34,7 @@ def _build_all_groups(self):
                 break
 
             location = Location(section_id=section_id)
-            for group in section_summary_context(location)["summary"]["groups"]:
+            for group in section_summary_context(location, "final_summary")["summary"][
+                "groups"
+            ]:
```

Comment:
Could we make the `"final_summary"` a keyword argument here, just for clarity and readability?. Threw me at first, then realised it was the `return_to` value.